TheTharz/TutorHub
# Login and Signup Functions Has Been Implemented

## Pull Request: Implement Sign In Navigation

### Changes Made:
- Added navigation logic for "Sign In" text in `LoginScreen`.
- Implemented the `navigateToSignIn` method to handle the navigation.

### Files Modified:
- `lib/features/authentication/screens/login/login.dart`
- `lib/features/authentication/screens/signup/signup_controller.dart`

### Details:
#### LoginScreen
- Added a `TapGestureRecognizer` for the "Sign In" text.
- Implemented the `navigateToSignIn` method for navigation.

#### SignUpController
- Modified the `signUp` method to handle internet connectivity, form validation, and privacy policy checks.
- Added success and error messages using `TLoaders`.
- Navigates to `LoginScreen` on successful sign-up.

### How to Test:
1. Run the app.
2. Navigate to the sign-up screen.
3. Fill in the required details and sign up.
4. Check for success message and navigate to the login screen.

### Checklist:
- [ ] Code follows project guidelines and style.
- [ ] Documentation and comments are updated.
